cmd/api/src/database/migration/migrations/schema.sql

```diff
@@ -70,6 +81,13 @@ CREATE SEQUENCE IF NOT EXISTS ad_data_quality_stats_id_seq
     CACHE 1;
 ALTER SEQUENCE ad_data_quality_stats_id_seq OWNED BY ad_data_quality_stats.id;
 
+CREATE TABLE IF NOT EXISTS analysis_request_switch (
+    singleton boolean DEFAULT true NOT NULL,
```

Comment:
I cannot find where this was technically set up as NOT NULL, should validate by comparing pg dumps

Reply:
PG dump comparison shows this is working as intended